frontend/src/pages/Landing/MyAlerts.js

```diff
@@ -142,10 +217,20 @@ function MyAlerts({ reports, newBtn }) {
 MyAlerts.propTypes = {
   reports: PropTypes.arrayOf(PropTypes.object),
   newBtn: PropTypes.bool.isRequired,
+  alertsSortConfig: PropTypes.shape({ sortBy: PropTypes.string, direction: PropTypes.string }),
+  alertsOffset: PropTypes.number,
+  alertsPerPage: PropTypes.number,
+  alertsActivePage: PropTypes.number,
+  alertReportsCount: PropTypes.number.isRequired,
+  sortHandler: PropTypes.func.isRequired,
 };
 
 MyAlerts.defaultProps = {
   reports: [],
+  alertsSortConfig: { sortBy: 'startDate', direction: 'asc' },
+  alertsOffset: 0,
+  alertsPerPage: 7,
```

Comment:
Should this default to 10?

Reply:
I changed it to 10, but right now since the pagination is not working it will be a placeholder only until we find a way to integrate the pagination with sorting (https://github.com/HHS/Head-Start-TTADP/issues/369)